apps/email/style/compose_cards.css

```diff
@@ -217,6 +216,8 @@ html:-moz-dir(rtl) .cmp-autocomplete-item-details {
   outline: 0;
   white-space: pre-wrap;
   word-wrap: break-word;
+  unicode-bidi: -moz-plaintext;
+  text-align: initial;
```

Comment:
Nearly every time we have a `unicode-bidi: plaintext`, I think we need a `text-align: initial. It is mainly for mail contents.

As in a browser with webpages, it is more natural to have an english email to be left-aligned, even if the user interface is in arabic. The reverse is also true.

plaintext also allows a per-paragraphe alignment: if you receive a mail with one English paragraph and one Arabic, each one will be aligned according to its logical `start`, ie left for English and right for Arabic. 

These two things are an exception to the rule 
> everything should be right-aligned in RTL, left-aligned in LTR

Some screenshots to show this:

## before
The arabic text in the mail is left-aligned in LTR, which does not feel ok.
LTR - RTL
![message_01](https://cloud.githubusercontent.com/assets/1576598/10695318/6aeeebb6-79a3-11e5-9968-f83d94767865.png)

Same problem in RTL for an English mail.
LTR - RTL
![message_02](https://cloud.githubusercontent.com/assets/1576598/10695365/9ec417ea-79a3-11e5-8366-10f9ffa05edf.png)


## After
In contrast, when we let the mail align naturally: 
![message_01](https://cloud.githubusercontent.com/assets/1576598/10695399/d8108df8-79a3-11e5-9308-b1328889335b.png)
![message_02](https://cloud.githubusercontent.com/assets/1576598/10695400/d8181794-79a3-11e5-8da2-ae09b6b35984.png)

Even in the Arabic interface, the English mail is correctly readable.

Notice that the bar indicating the quote DOES mirror, because it is more part of the chrome than the mail (as we parse the `>` and format them). Likewise, the subject is still aligned according to the interface, also because the date juste below is.